components/fetchCampusLocation/FetchCampusLocation.component.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import {
+  StyleSheet,
+  View,
+  Text,
+  Dimensions,
+  TouchableHighlight
+} from "react-native";
+
+const FetchCampusLocation = props => {
+  /**
+   * this hook sets a boolean variable isSGW to know which current
+   * button is pressed
+   */
+  const [isSGW, setIsSGW] = useState(true);
+
+  /**
+   * Changes the boolean value and the coordinates that will change
+   * the map view
+   */
+  const onSGWPressButton = () => {
+    setIsSGW(true);
+    props.regionChange({
+      latitude: 45.495869,
```

Comment:
These hardcoded values should be stored in a separate const.js file somewhere. Leo was doing a similar thing. Please discuss with him

Reply:
1- Jonathan said something about doing it later
2- I discussed the const file with Leo before and we are wondering what the database is for and if we should store it there